### PR TITLE
Project : Facial Expression Recognition with PyTorch Issue #697

### PR DESCRIPTION
**Pull request by revanth1718**
Resolves Issue: #697 

✔️ Involved training neural networks to accurately classify facial expressions using annotated facial image datasets, models were constructed and trained with PyTorch's deep learning capabilities.
✔️ Project was employed by convolutional neural networks (CNNs).
✔️ The success of the project was gauged through the model's proficiency in accurately identifying expressions on unseen data, with potential applications ranging from emotion-aware interfaces to behavioral analysis in psychology and market research.

Screenshot (550)
I am Participating in GSSOC'24. Please Label this Project as Level 3.

### CODE REVIEW

**Comment by Niketkumardheeryan** [comment]
hey @revanth1718, there are 51 commits ,please create a new PR , as there is lots of confusion because of this.closing this one.
